libs/coin-modules/coin-cosmos/src/chain/Mantra.ts

```diff
@@ -0,0 +1,20 @@
+import CosmosBase from "./cosmosBase";
+
+class Mantra extends CosmosBase {
+  stakingDocUrl: string;
+  unbondingPeriod: number;
+  prefix: string;
+  validatorPrefix: string;
+  // Provided by coin config
+  ledgerValidator!: string;
+  lcd!: string;
+  constructor() {
+    super();
+    this.stakingDocUrl = "";
```

Comment:
We need to add the doc url here

Reply:
https://github.com/LedgerHQ/ledger-live/pull/8205/files#diff-ac58aed97bc6283523a1a1299b2f73d88100f93300cb1027cbc61709934f01c6R13